ff_stdio.c

```diff
@@ -1137,9 +1137,9 @@ int ff_stat( const char * pcName,
 
         #if ( ffconfigTIME_SUPPORT == 1 )
             {
-                pxStatBuffer->st_atime = prvFileTime( &( xDirEntry.xAccessedTime ) );
-                pxStatBuffer->st_mtime = prvFileTime( &( xDirEntry.xModifiedTime ) );
-                pxStatBuffer->st_ctime = prvFileTime( &( xDirEntry.xCreateTime ) );
+                pxStatBuffer->ff_atime = prvFileTime( &( xDirEntry.xAccessedTime ) );
+                pxStatBuffer->ff_mtime = prvFileTime( &( xDirEntry.xModifiedTime ) );
+                pxStatBuffer->ff_ctime = prvFileTime( &( xDirEntry.xCreateTime ) );
```

Comment:
Same here. Would this break existing users code?

Reply:
Can you please have a look at my [earlier response](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-FAT/issues/50#issuecomment-1672806711)?

Reply:
This would only break existing user code if they are accessing the values inside the pxStatBuffer in their code in some way associated with `st_[a|m|c]time`.  It can be mitigated by providing users a description of the breaking change in this `Lab-Project` based repo.

The other option is to possibly investigate replacing the entire struct for Linux based examples with the linux `stat` structure instead of using home-grown version, and ensuring compatibility between the FF_Stat and linux `stat` is maintained - see https://github.com/FreeRTOS/Lab-Project-FreeRTOS-FAT/issues/50#issuecomment-1675726285.